tests/behat/features/asp/ondemand-unauth.feature

```diff
@@ -27,9 +27,10 @@ Feature: This Behat test checks the OnDemand page
     When I go to "/ondemand"
     Then I should see "Bridges2"
     Then I should see an image with alt text "Bridges2 logo"
-    When I click "Bridges2"
```

Comment:
This failure is frustrating.  It's been a chronic problem, that I have repeatedly tried to get around.  Not sure why it happens.